modules/benchmark/R/load_data.R

```diff
@@ -26,13 +27,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2)) { 
+    #To Do: call to DAP to see if conversion to csv is possible
+    #Brown Dog API call through BDFiddle, requires username and password
+    #will be made into R package
+    install_github("yan130/bd.r")
```

Comment:
Agree 100%. Also, you don't want to re-install the package every time you call the function.